assets/js/app/DataLayers/intervals.js

```diff
@@ -186,8 +191,8 @@ LocusZoom.DataLayers.add("intervals", function(layout){
                     .data([interval], function(d){ return d[data_layer.layout.id_field] + "_bbox"; });
                 if (data_layer.layout.split_tracks){
                     bboxes.remove();
-                } else {                    
-                    bboxes.enter().append("rect")
+                } else {
+                    bboxes.enter().insert("rect", ":first-child")
```

Comment:
This was a good find! In #106 you refer to this as a likely regression; perhaps a unit/regression test could help prevent it from happening again?

Reply:
Ideally sure, more automated testing is generally a good thing. In practice, though, it would require automated testing against the composition of the SVG which is more of an integration/behavior level test, not a unit test. We could do it in mocha but it would likely be brittle.

Establishing a pattern for robust integration tests in LocusZoom is a would-be-nice that we may not have the resources for at the moment.